lib/fs.js

```diff
@@ -109,9 +109,10 @@ const {
   parseFileMode,
   validateBuffer,
   validateInteger,
-  validateInt32,
-  validateUint32
+  validateInt32
 } = require('internal/validators');
+// 2 ** 32 - 1
+const kMaxId = 4294967295;
```

Comment:
Maybe `kMaxUid` (or otherwise `kMaxUserId`)? `kMaxId` is *very* generic…

Reply:
`kMaxUserId` i think🤔